cheatsheets/Cross-Site_Request_Forgery_Prevention_Cheat_Sheet.md

```diff
@@ -127,7 +127,7 @@ We recommend researching if the framework you are using has an option to achieve
 
 Most developers tend to ignore CSRF vulnerability on login forms as they assume that CSRF would not be applicable on login forms because user is not authenticated at that stage. That assumption is false. CSRF vulnerability can still occur on login forms where the user is not authenticated, but the impact/risk view for it is quite different from the impact/risk view of a general CSRF vulnerability (when a user is authenticated).
 
-With a CSRF vulnerability on login form, an attacker can make a victim login as them and learn behavior from their searches. For more information about login CSRF and other risks, see section 3 of [this](https://seclab.stanford.edu/websec/csrf/csrf.pdf) paper.
+With a CSRF vulnerability on login form, an attacker can make a victim login with an account they control. The attacker can then learn behaviour from their interaction with the site with features such as search/activity/navigation history. The victim might also end up trusting the site to perform operations such as providing highly confidential information such as payment details assuming that they are logged into their account. For more information about login CSRF and it's associated risks, see section 3 of [this](https://seclab.stanford.edu/websec/csrf/csrf.pdf) paper.
```

Comment:
As I have spotted several English mistakes, I thought of refactoring it for user readability. Let me know what you think:

Having a CSRF vulnerability on the login form, an attacker can lure a victim to login with an account that the attacker controls. The user’s behavior would then be studied by the actions taken, and if that application contains highly confidential information (_e.g._ payment details), those too will be compromised. For additional information on this attack, refer to this [paper](https://seclab.stanford.edu/websec/csrf/csrf.pdf).